### ISSUE TITLE
Problems showing art and game details for several GBA and SNES games

### ISSUE THREAD

**Issue by mikerealohs** [mmatyas/pegasus-frontend#745]
### The issue
I'm having issues showing the associated game art and metadata info for several games on two systems. While these games open ok through Pegasus when selected, they do not show their respective art and metadata (after several app upgrades, of both the stable and latest versions)... The titles affected by this bug are listed below:

GBA:
Godzilla Domination!
Kirby: Nightmare in Dreamland
Pokemon Pinball: Ruby and Sapphire
The Revenge of Shinobi

SNES:
Donkey Kong Country 2: Diddy's Kong Quest
The Legend of Zelda: A Link to the Past

Instead of showing the appropriate art, title and metadata for each game in the respective place under their associated console, Pegasus shows a blank space with the Roms titles in place holder text (FYI, every other game included in each of those system folders displays art and metadata correctly). 

### System info

Retroid Pocket 2 running Android version 6.0 and Build version 202010212012

### Pegasus version

Pegasus alpha14-73-g0c109791 (2020-11-28)

### Pegasus log

[lastrun.log](https://github.com/mmatyas/pegasus-frontend/files/5624938/lastrun.log)
[metadata.txt](https://github.com/mmatyas/pegasus-frontend/files/5624948/metadata.txt)
[metadata.txt](https://github.com/mmatyas/pegasus-frontend/files/5624977/metadata.txt)

**Comment by mmatyas** [comment]
Hi! I've looked into the log and the metafiles, and according to the log the files set in the metadata seem to be missing:

```
[w] Metafiles: `/mnt/media_rw/1234567890/Roms/GBA/metadata.txt`, line 619: Game file `/mnt/media_rw/1234567890/Roms/GBA/Pokemon Pinball - Ruby & Sapphire.gba` doesn't seem to exist
[w] Metafiles: `/mnt/media_rw/1234567890/Roms/GBA/metadata.txt`, line 1361: Game file `/mnt/media_rw/1234567890/Roms/GBA/Godzilla - Domination!.gba` doesn't seem to exist
[w] Metafiles: `/mnt/media_rw/1234567890/Roms/GBA/metadata.txt`, line 2180: Game file `/mnt/media_rw/1234567890/Roms/GBA/Kirby - Nightmare In Dream Land.gba` doesn't seem to exist
[w] Metafiles: `/mnt/media_rw/1234567890/Roms/SNES/metadata.txt`, line 55: Game file `/mnt/media_rw/1234567890/Roms/SNES/Donkey Kong Country 2 - Diddy's Kong Quest.smc` doesn't seem to exist
[w] Metafiles: `/mnt/media_rw/1234567890/Roms/SNES/metadata.txt`, line 1147: Game file `/mnt/media_rw/1234567890/Roms/SNES/The Legend of Zelda - A Link to the Past.smc` doesn't seem to exist
```

which results the 

Since you mentioned that they launch fine, perhaps there's a typo in the metadata file? I see all games have a `-` in their filenames, for example, is that the correct name? Ie. is it `Godzilla - Domination!.gba` and not `Godzilla Domination!.gba`?

`The Revenge of Shinobi` seems to be an interesting case, but let's return to that once the rest is fixed.

**Comment by mikerealohs** [comment]
Hey, So I saw the same thing, and looked at the metadata file and compared them to the file names (and they're correct) prior to submitting the bug request...

Further like you, I too thought it had something to do with the inclusion of the hyphens, but their are other games with hyphens in my metadata file/Rom folder (plus not all of the games with issues have hyphens?!?).... Further complicating this/confusing me is the fact that the names match in the metadata file and as named in the associated Rom folder (I built the metadata file using Skraper and the files as I had them in the folder on my PC, so they should, and do match)...

**Comment by mikerealohs** [comment]
OK, I stand corrected. You were correct, this is in fact a user issue... 🤦‍♂️  The issue lied with inconsistent file names on the SD card _in my Retroid_ vs the metadata file.

I standardized and corrected all of my game file names (after initially saving them to my SD card in the Retroid) and before scraping the files. Instead of wiping the SD card and dragging and dropping the newly corrected files with names fully matching the metadata in their respective Rom folders (I must screwed this up on these two systems when setting up Pegasus)...

Anywho, long story short I was only looking at the main folder on my computer which had the correct file names (that I'd used to build the metadata files). I've since corrected the file names with issues to IDENTICALLY match the metadata file (they had spacing and punctuation inconsistencies), and that's resolved the issues with these Roms, Apologies and thanks for the handholding on this (admittedly self made) issue... 🤦‍♂️
